static/skin/i18n/en.json

```diff
@@ -28,7 +28,7 @@
 	, "random-page-button-text": "Go to a randomly selected page"
 	, "searchbox-tooltip": "Search '{{BOOK_TITLE}}'"
 	, "confusion-of-tongues": "Two or more books in different languages would participate in search, which may lead to confusing results."
-	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"?lang=\">reset filter</a>?"
+	, "welcome-page-overzealous-filter": "No result. Would you like to <a href=\"#lang=\">reset filter</a>?"
```

Comment:
This is not really related to this change but why specifying `lang=` ? If we don't want a filter, shouldn't we pass no arguments at all ?

Reply:
If we do that, the arguments will be taken from cookie values and it'll be on the same no results page everytime

Reply:
Ok, let's move on is it is for this PR. 
But it somehow rejoin the discussion in https://github.com/kiwix/libkiwix/pull/897#discussion_r1129196868, let's continue the discussion there.